File: Bimestre 2/Python/Arreglos/Ascendente.py
aux=0;n=0;
print("Ingrese el tamaño del vector")
n=int(input())
#declara vector
A = []
print("Ingrese los terminos del vector")

for i in range(n):
    print("A[",i,"]", end="")
    num=int(input())
    A.append(num)
#Presentar vector original
print("Vector original")
for i in range(n):
    print("A[",i,"]=",A[i], end="")

#ordenar vector
for i in range(n):
    for j in range(n):
        if(A[i]>A[j]):
            aux=A[i]
            A[i]=A[j]
            A[j]=aux

#Presentar el vector ordenado
print("---VECTOR ORDENADO-----")
for i in range(n):
    print("A[",i,"]=",A[i])

File: Bimestre 2/Java/Arreglos/src/arreglos/Ascendente.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package arreglos;

import java.util.Scanner;

/**
 *
 * @author diegojp
 */
public class Ascendente {
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
        int n,i,j,aux;
        aux=0;n=0;
        System.out.println("Ingrese el tamaño del vector");
        n=leer.nextInt();
        //declara vector
        int [] A = new int [n];
        
        System.out.println("Ingrese los terminos del vector");
        
        for(i=0; i<n; i++){
            System.out.println("A["+i+"]");
            A[i]=leer.nextInt();
        }
        //Presentar vector original
        System.out.println("Vector original");
        for(i=0; i<n; i++){
            System.out.println("A["+i+"]="+A[i]);
        }
        
        //ordenar vector
        for(i=0; i<n; i++){
            for(j=i+1; j<n; j++){
                if(A[i]>A[j]){
                    aux=A[i];
                    A[i]=A[j];
                    A[j]=aux;
                }
            }
        }
        //Presentar el vector ordenado
        System.out.println("---VECTOR ORDENADO-----");
        for(i=0; i<n; i++){
            System.out.println("A["+i+"]="+A[i]);
        }
    }
    
}
